bin/oneoff/generate_legacy_survey_summaries.rb

```diff
@@ -131,15 +131,17 @@ def snapshot_summer_workshops_from_jotform(course)
     facilitator_report = {}
 
     unless all_their_completed_workshops.empty?
-      rollup = report_facilitator_rollup(id, all_their_completed_workshops.first, false)
-
-      key = "facilitator_#{id}_all_ws"
-      rollup[:rollups][key][:averages].each do |string_key, average|
-        # We have found a string_key and an average, now find out the actual string for the key
-        question_text = rollup[:questions][string_key]
-        unless question_text.nil?
-          question_text.gsub!("{facilitatorName}", f.name)
-          facilitator_report[question_text] = average
+      # Do workshop rollups, then facilitator rollups.
+      [false, true].each do |facilitator_rollups|
```

Comment:
nit: can you make the facilitator_rollups variable name more clear? Ex only_facilitator_questions?

Reply:
Sure thing.  

I did struggle over this particular issue.  The comment above is perhaps just as useful: it borrows the facilitator- and workhop-rollups naming from these lines: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/lib/pd/survey_pipeline/survey_pipeline_helper.rb#L36-L37

It's one challenge of using a boolean parameter to actually switch between different modes.

Reply:
thanks! the comment was definitely helpful, without that I would have been very confused :)